types/absolute-url/package.json

```diff
@@ -8,7 +8,7 @@
     ],
     "dependencies": {
         "@types/express": "*",
-        "@types/express-serve-static-core": "^4.17.33"
```

Comment:
the original dependency declaration leads to discrepancy between `express` (resolved as 5) and `express-serve-static-core` (resolved as 4). so that altering `Request` for assigning additional property no longer works.

Reply:
alternatively, `@types/express` can be set to `^4`

Reply:
- please advise how to address this issue better.
- should I change the version of types to `2.1.9999` even if the package itself is `2.0` latest?

Reply:
Thanks, `*` is good. I probably shouldn't have restricted that in the first place.

I wouldn't bump the version. You'd only do that if you machted some new features in the package itself.

Reply:
OK. Keeping this change as is then.
Thank you @tpluscode